src/server/api/schema.js

```diff
@@ -578,7 +559,7 @@ const rootMutations = {
       let createCannedResponses = r
         .knex('canned_response')
         .where({ campaign_id: oldCampaignId })
-        .then(function (res) {
+        .then(function(res) {
```

Comment:
if you're changing white space might as well fix it the way lint would ask and use `(res) => {`...